avalon/io.py

```diff
@@ -56,7 +56,10 @@ def install():
 
     timeout = int(Session["AVALON_TIMEOUT"])
     self._mongo_client = pymongo.MongoClient(
-        Session["AVALON_MONGO"], serverSelectionTimeoutMS=timeout)
+        host=Session["AVALON_MONGO_HOST"],
```

Comment:
Is really required to have split host and url?

Reply:
Yeah, as far as my testing went. Please have a go yourself, cause I did this in a hurry so I may not have tested thoroughly enough.